iDiff/utils/output_templates/multiVersionOutput.txt

```diff
@@ -1,6 +1,10 @@
-Packages found only in {{.Diff.Image1}}:{{range $name, $value := .Diff.Packages1}}{{"\n"}}{{print "-"}}{{$name}}{{end}}{{"\n"}}
-Packages found only in {{.Diff.Image2}}:{{range $name, $value := .Diff.Packages2}}{{"\n"}}{{print "-"}}{{$name}}{{end}}
+-----{{.DiffType}}-----
 
-Version differences:{{"\n"}}	(Package:	{{.Diff.Image1}}{{"\t\t"}}{{.Diff.Image2}}){{range .Diff.InfoDiff}}
-	{{.Package}}:	{{range .Info1}}{{.Version}}{{end}}	{{"\t"}} {{range .Info2}}{{.Version}}{{end}}
-	{{end}}
+Packages found only in {{.Diff.Image1}}:{{ if not .Diff.Packages1 }} No unique packages in Image1{{ else }}{{range $name, $value := .Diff.Packages1}}{{"\n"}}{{print "-"}}{{$name}}{{end}}{{ end }}
```

Comment:
When I use the image `gcr.io/google-appengine/python:staging`, the output from the tool looks like this:

```
Packages found only in pythonlatest: No unique packages in Image1
```
The naming of the image is a bit cluttered: could we at least keep the colon between the image name and the tag? I would actually even prefer to just have the entire image path in there, just for clarity.

Also, I think "No unique packages in Image1" is a little verbose, especially when it appears on the screen 4 or 6 times. Could we just change that to "None", since it's already clear what that's referring to by the "Packages found only in <image>:"?